ReaderApp.js

```diff
@@ -1686,19 +1703,12 @@ class ReaderApp extends React.PureComponent {
           }),
         }, () => {
           if (data.aggregations) {
-            let availableFilters = [];
-            let registry = {};
-            let orphans = [];
-            for (let aggregation of aggregation_field_array) {
-              if (!!data.aggregations[aggregation]) {
-                const { buckets } = data.aggregations[aggregation];
-                const { availableFilters: tempAvailable, registry: tempRegistry, orphans: tempOrphans } = Sefaria.search[build_and_apply_filters](buckets, appliedFilters, appliedFilterAggTypes, aggregation, Sefaria);
-                availableFilters.push(...tempAvailable);  // array concat
-                registry = {...registry, ...tempRegistry};
-                orphans.push(...tempOrphans);
-                this.setAvailableSearchFilters(type, availableFilters, orphans);
-              }
-            }
+            updateAggregations(data);
+          } else if (getFilters) {
+            [queryProps.appliedFilterAggTypes, queryProps.applied_filters, queryProps.aggregationsToUpdate] = [[], [], ['path']];
```

Comment:
why is aggregationsToUpdate hard-coded to 'path'? I don't believe this works for sheet search because there are different filters there.